tests/tests/xfstests.sh

```diff
@@ -64,6 +64,8 @@ generic/029	# mmap missing
 generic/030	# mmap missing
 generic/075	# file content mismatch failures (fds, etc)
 generic/080	# mmap missing
+generic/091	# skip fsx tests
+generic/094	# odirect streaming pre-alloc treated as failure in xfstests
```

Comment:
You know, I bet if we change the dio alloc_blocks to only allocate the size of the IO it'll fix these tests that were so cranky about our preallocation.  It'd be worth trying them again once that's in place.

Reply:
Still no go on this one :/